tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -165,7 +164,7 @@ public function deleteAlias(string $reference): void
         $aliasId = $this->getSharedStorage()->get($reference);
 
         try {
-            $this->getCommandBus()->handle(new DeleteAliasCommand(new AliasId((int) $aliasId)));
+            $this->getCommandBus()->handle(new DeleteAliasCommand((int) $aliasId));
```

Comment:
you should save the aliasId as INT already in shared storage, so this casting should not be needed.